src/main/java/lotto/domain/ticket/LottoTickets.java

```diff
@@ -15,6 +16,11 @@ public LottoTickets(List<LottoTicket> lottoTickets) {
         this.lottoTickets = lottoTickets;
     }
 
+    public LottoTickets(LottoTickets lottoTickets, LottoTickets joinLottoTickets) {
+        this.lottoTickets = lottoTickets.getLottoTickets();
+        this.lottoTickets.addAll(joinLottoTickets.getLottoTickets());
+    }
```

Comment:
```suggestion
    public LottoTickets(LottoTickets lottoTickets, LottoTickets joinLottoTickets) {
        this.lottoTickets = lottoTickets.lottoTickets;
        this.lottoTickets.addAll(joinLottoTickets.lottoTickets);
    }
```
이러면 getter를 사용하지 않을 수 있습니다!